poppler-sys/README.md

```diff
@@ -5,9 +5,9 @@ Low level bindings to [poppler](https://gitlab.freedesktop.org/poppler/poppler).
 ## Vendored Binding
 
 The vendored bindings were generated on a system which had headers from:
-- poppler-glib 0.76.1
-- glib-2.0 2.60.4
-- cairo 1.16.0
+- poppler-glib 0.87.0-1
```

Comment:
I haven't checked if there is any meaningful difference, but on my machine I only had the `0.87.0-1` version (instead of `0.87.0`).